rmgpy/statmech/rotationTest.py

```diff
@@ -32,6 +32,10 @@
 This script contains unit tests of the :mod:`rmgpy.statmech.rotation` module.
 """
 
+from __future__ import division
+from future import standard_library
```

Comment:
I'm unfamiliar with this, could you teach me what it does?

Reply:
There are certain functions which were renamed in Python 3. This (and the line below) makes it so that you can use the Python 3 names and get the correct behavior.

In this specific case, it's because pickle/cPickle were combined into pickle in Python 3, which automatically decides whether to use the Python or C implementation.